api/v1alpha1/addresspool_types.go

```diff
@@ -42,10 +42,19 @@ type AddressPoolSpec struct {
 	AutoAssign bool `json:"autoAssign,omitempty"`
 }
 
+// AddressPool Services information
+type AddressPoolSvcStats struct {
+	// Service Name
+	Service string `json:"service,omitempty"`
+
+	// Service's Allocated Externat IP address
+	ExternalIP string `json:"external-ip,omitempty"`
+}
+
 // AddressPoolStatus defines the observed state of AddressPool
 type AddressPoolStatus struct {
-	// INSERT ADDITIONAL STATUS FIELD - define observed state of cluster
-	// Important: Run "make" to regenerate code after modifying this file
+	AddressPoolServices []AddressPoolSvcStats `json:"address-pool-services"`
+	TotalAllocatedIps   int                   `json:"total-allocated-ips,omitempty"`
```

Comment:
this one should be I think `totalAllocatedIPs`

Reply:
same as above it has to be Capital letter